apps/website/.vuepress/theme/global-components/DocAlert.vue

```diff
@@ -11,6 +14,7 @@ export default {
   name: 'DocAlert',
   props: {
     status: { type: String, default: 'info' },
+    path: { type: String, defualt: null },
```

Comment:
`default` instead of `defualt`

Reply:
So 🎅 brought me a new keyboard that I am still getting used to 🤫 ....
Will fix.